packages/express/index.ts

```diff
@@ -0,0 +1,50 @@
+import type { RequestHandler } from 'express'
+import { Router } from 'express'
+import type { KopflosConfig } from '@kopflos-cms/core'
+import absolutUrl from 'absolute-url'
+import Kopflos from '@kopflos-cms/core'
+import rdfHandler from '@rdfjs/express-handler'
+import factory from '@zazuko/env-node'
+import onetime from 'onetime'
```

Comment:
onetime and once are similar?

Reply:
`onetime` is ESM
Also, according to onetime's author: `Unlike the module [once](https://github.com/isaacs/once), this one isn't naughty and extending Function.prototype.`